templates/components/form/link_existing_or_new.html.twig

```diff
@@ -74,7 +74,7 @@
          </div>
          <div class="col-auto ms-4">
             {% if create_link %}
-               {% set target_form_path = (target_itemtype|itemtype_form_path ~ "?_" ~ source_itemtype|itemtype_foreign_key ~ "=" ~ source_items_id) %}
+               {% set target_form_path = (target_itemtype|itemtype_form_path ~ "?" ~ source_itemtype|itemtype_foreign_key ~ "=" ~ source_items_id) %}
```

Comment:
When creating a ticket from a project task, the foreign key needs the underscore in front. That seems to be the only case, and must have been the first case I migrated. Perhaps the code in the Ticket class should be changed to handle both `projecttasks_id` and `_projecttasks_id` in addition to this change.